### ISSUE TITLE
Change excluded repositories list

### ISSUE THREAD

**Issue by iamale** [cla-assistant/cla-assistant#182]
In case new repositories are added that aren't covered by the CLA etc.

**Comment by KharitonOff** [comment]
Good point. You are very welcome to contribute to CLA assistant 😄 
Otherwise the workaround would be to relink your org every time you want to change exclude pattern.

**Comment by iamale** [comment]
@KharitonOff I'll look into this, probably this weekend. The app looks quite complex to me, though, and it would be nice if you pointed me where to start :)

**Comment by KharitonOff** [comment]
Sure, I can support you on this.
The main view of the CLA assistant app is implemented in 
/src/client/templates/home.html and corresponding controller is /src/client/controller/home.js
each item of the linked repo / org is handled in
/src/client/templates/settings.html and corresponding controller is /src/client/controller/settings.js

The settings view/controller would probably the best place where you can add logic for changing the pattern.

We use $RPCService for backend calls. It's method "call" has following signature:
 - obj - corresponds to the filenames stored in /src/server/api (e.g. org -> org.js)
 - fun - corresponds to the method of the api
 - data - JSON data that should be sent via POST request. They are stored in req.args attribute on the server side.

You will probably need to call 
```
$RPCService('org', 'update', {
                orgId: $scope.selected.item.id,
                org: $scope.selected.item.login,
                gist: $scope.selected.gist.url,
                excludePattern: $scope.selected.item.excludePattern
            });
```
!!! But this function doesn't exist yet 😄 

So on the backend we need "update" function for /src/server/api/org.js which will call (also newly created!) update function of /src/server/services/org.js where you can update the corresponding mongoose data entry.

I hope it's not to much work for you. If you need some help just open a WIP-PR and I'll be glad to help you on that.

You can also take a look on the PR where this feature was implemented: #162
